extensions/blocks/podcast-player/style.scss

```diff
@@ -62,6 +60,13 @@ $player-float-background: $light-gray-200;
 	.jetpack-podcast-player {
 		padding-top: 0;
 		padding-bottom: 0;
+
+		/**
+		* Set default values for our CSS variables.
+		*/
+		--jetpack-podcast-player-primary: #{$jetpack-podcast-player-primary};
+		--jetpack-podcast-player-secondary: #{$jetpack-podcast-player-secondary};
+		--jetpack-podcast-player-background: #{$jetpack-podcast-player-background};
```

Comment:
I think it would be good to have few comments here about:
1. Why we're prefixing only these with `jetpack-podcast-player-*` and
2. Why we're using CSS vars only for some elements.